File: backend/src/main/java/cohappy/backend/model/dto/request/PatchChoreDTO.java
package cohappy.backend.model.dto.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PatchChoreDTO {
    private String choreCode;
    private LocalDate day;
    private String assignedTo;
    private Boolean completed;
    private String houseCode;
    private String name;
    private String description;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/PatchChoreDTO.kt
package cohappy.frontend.model.dto.request;

import java.time.LocalDate;

data class PatchChoreDTO(
    val choreCode: String,
    val day: LocalDate,
    var assignedTo: String? = null,
    var completed: Boolean? = null,
    var houseCode: String? = null,
    var name: String? = null,
    var description: String? = null
)
